### ISSUE TITLE
Request: REST api

### ISSUE THREAD

**Issue by ojasvinaik13** [simple-icons/simple-icons#10841]
### Brand Name

REST API

### Website

https://restfulapi.net/

### Popularity Metric

The restfulapi.net rank is #341,327. See https://www.similarweb.com/website/restfulapi.net/#overview 

### Official Resources for Icon and Color

None

**Comment by service-paradis** [comment]
REST is not really a brand and does not really have any icons. I dont think we can really add this.

**Comment by uncenter** [comment]
I agree. The website linked is a guide/tutorial and besides not having any logos it isn't a brand or company or product and therefore doesn't really seem fit for inclusion under our guidelines.
